### ISSUE TITLE
Help with unam binder and crypto miner.

### ISSUE THREAD

**Issue by kqxmii** [UnamSanctam/SilentCryptoMiner#2199]
Okay so i have a folder. inside the folder, there is
📁 ENGINE
      > ▶️ GPShell.exe (Crypto Miner)
📄GPShell.py
▶️ Test.exe
▶️ BindedTest.exe

so, my test.exe is a simple app/ui right. and GPShell.py is a script that when it's run, it opens GPShell.**exe**. I then binded Test.exe and GPShell.py, so that when i run test.exe, it opens the app, and also opens GPShell.**exe**, which is the miner. thing is, before i binded it, it all worked fine (except i was using a .txt instead of a .exe inside the engine folder). since that worked, i replaced the .txt and changed the code so it opens GPShell.exe, and then i binded test and gpshell.py. but, when i run the bindedtest.exe, it doesnt run gpshell. i even use the gpshell-checker.exe and it says not running. help??

(code for my .py file if needed)
```
import subprocess
import os

def open_GPShell():
    # Get the directory where the Python script is located
    script_dir = os.path.dirname(os.path.abspath(__file__))
    
    # Assume the "engine" folder is in the same directory as GPShell
    engine_path = os.path.join(script_dir, "ENGINE")
    GPShell_exe_path = os.path.join(engine_path, "GPShell.exe")
    
    if os.path.exists(GPShell_exe_path):
        subprocess.Popen([GPShell_exe_path], creationflags=subprocess.CREATE_NO_WINDOW)

if __name__ == "__main__":
    open_GPShell()

```

**Comment by kqxmii** [comment]
also, i basically jus want to make it so if i run the .exe, it also runs GPShell. i tried to bind gpshell and the .exe app together, but when i ran it, the .exe app didnt open.

**Comment by UnamSanctam** [comment]
If you're using UnamBinder then you should be able to just bind GPShell.exe and the other .exe app together directly.

**Comment by kqxmii** [comment]
alright, but lets say i run the file to test it, and it runs the crypto miner. how do i uninstall the crypto miner? is it possible to do so after i binded it and then ran it?

**Comment by UnamSanctam** [comment]
You can use the uninstaller that gets built together with the miner, a bound program behaves no differently than an unbound one.

**Comment by kqxmii** [comment]
does the uninstaller have to be in the same folder?

**Comment by UnamSanctam** [comment]
No it can be anywhere on the computer.

**Comment by kqxmii** [comment]
Okay so, i binded the normal app with the crypto miner. now when i run the app, it opens 2 windows of the app, and doesnt open the crypto miner at all. it seems that binding the crypto miner makes it not work.

**Comment by UnamSanctam** [comment]
>Okay so, i binded the normal app with the crypto miner. now when i run the app, it opens 2 windows of the app, and doesnt open the crypto miner at all. it seems that binding the crypto miner makes it not work.

Are you sure you didn't bind the app twice?

**Comment by kqxmii** [comment]
Oh, both the files had the same filename. is that why?

**Comment by UnamSanctam** [comment]
Yes, in that case one of them will overwrite the other and then when it runs the files it will run the same file twice because both have the same name.
